src/components/Calendar/CalendarRoot.tsx

```diff
@@ -223,6 +224,18 @@ class ScheduleCalendar extends PureComponent<ScheduleCalendarProps, ScheduleCale
         canvas.style.color = 'black';
         calendarHeader.style.marginRight = '0px';
 
+        // Set each rbc header and time slot text color to white if the theme is dark
```

Comment:
I think you should change the contents of this comment. The code is pretty self-explanatory so you don't need to explain _what_ it's doing. However, I have no clue _why_ we're changing these properties of rbc_headers and rbc_time_slots, or even where in the DOM they are. The comment should explain the _why_ of this block, not the _what_.